context/app/static/js/components/Detail/DerivedDatasetsSection/DerivedDatasetsSection.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import DerivedEntitiesSectionWrapper from 'js/components/Detail/DerivedEntitiesSectionWrapper';
+import DerivedEntitiesTable from 'js/components/Detail/DerivedEntitiesTable';
+import DerivedEntitiesSectionHeader from 'js/components/Detail/DerivedEntitiesSectionHeader';
```

Comment:
In general, do we want to avoid paths where higher directories are capitalized? ie, a directory of reusable components shouldn't also be a component in it's own right? But... I'm not sure that should be a hard-and-fast rule ... and I think just renaming the directory to `details` would cause problems because the the Mac FS is case insensitive?

Reply:
I think we'd like to change it to `detail` but doing so right now is more trouble than it is worth?